scripts/ansible-runner

```diff
@@ -45,119 +45,27 @@ usage()
 }
 if [ -n "$1" ]; then usage; fi
 
+source ansible-common.sh
```

Comment:
same as above

did you tested it from another directory ?
I want to make sure this will still work if you run for example from /opt/concoursevolume/ the script using /usr/..../ansible-cli